tests/integration/s2n_client_endpoint_handshake_test.py

```diff
@@ -114,15 +122,22 @@ def well_known_endpoints_test(use_corked_io):
 
 def main(argv):
     if len(argv) < 2:
-        print("s2n_handshake_test_s_client.py host port")
+        print("s2n_client_endpoint_handshake_test.py host port")
         sys.exit(1)
 
     host = argv[0]
     port = argv[1]
 
     failed = 0
-    failed += well_known_endpoints_test(False)
-    failed += well_known_endpoints_test(True)
+
+    # TLS 1.2 Tests
+    failed += well_known_endpoints_test(False, False)
```

Comment:
the argument list is starting to become unreadable when more options get passed into it into the function, although this might still be manageable. If this becomes more definitely pass a dict or an object instead.

Reply:
Even with two arguments you could use named parameters. This will make the behavior more explicit when somebody reads the tests.

```
def well_known_endpoints_test(use_corked_io=False, tls13_enabled=False):
    # pass


failed += well_known_endpoints_test()
failed += well_known_endpoints_test(tls13_enabled=True)
```

Reply:
Fair point! I added named parameters to the latest version.